culture.md

```diff
@@ -18,15 +18,14 @@ There are a huge number of channels, over 250 at time of this writing. A great m
 
 We support autonomy: most channels were simply created by someone who decided it was a good idea. We frown on doing things to people without their consent.
 
-Some channels have a `-unfiltered` or `-explicit` counterpart; it's completely acceptable to suggest moving a conversation to the `-unfiltered` channel, and completely normal to leave that channel if upsetting conversation is there. This leaves identity channels free to keep their membership so people can find each other, without having to be in conversations they are not equipped for. We've worked on making sure it's acceptable to leave channels for any reason without it being implied that someone speaking is a problem. Hard topics happen, are encouraged, and space has to be made for them, but we've also tried to make sure that there is space for self care and that it's okay to leave a conversation. Sometimes more sexually explicit conversation will spring up in these less filtered spaces.
+Someone often requests that someone redirect a conversation into another channel or thread. This should usually get honored without much discussion. Threads leave a great place to give content warnings about touchy subject matter, or discuss in more explicit detail while letting people opt in, rather than having graphic, disturbing or otherwise upsetting things in the middle of otherwise enjoyable space for them.
```

Comment:
I would suggest adding "Please consider editing the thread root with appropriate content warnings if the thread content ends up containing sensitive subject matter that wouldn't otherwise be obvious."

Reply:
Thank you! Amended!